docs/examples/simple-webapp-auth/README.md

```diff
@@ -0,0 +1,125 @@
+The credential subheaders may seem a little confusing. In essence, `Name` refers to the name of the application you need credentials for. In our case, it's the simple WebApp. The `Key` is how the application then queries its credentials with `Value` referring to the actual contents of the key (the secure bit). 
+For our example, add the following credential:
+
+```
+Name = my_simple_web_app
+Key = web_app_credentials
+Value = username:password
+
+```
+
+In place of `username:password` you can enter your own username and password but it is sufficient for this demonstration to leave it as is. 
+
+The credentials are now ready to be used on all your WoTT enabled devices. On your server device, you will now need to relaunch the WoTT Agent to download the new credentials:
+
+``` 
+$ sudo service wott-agent restart
+
+```
+There will now be a JSON file in your WoTT agent's credentials with your information which the app will parse and process. 
+
+**Note:** If you change the Name of the new credentials, you will need to edit the name of the JSON file being read within the app as it currently assumes you have named it `my_simple_web_app` as per instructions.
+
+
+## Setting up the WebApp
+
+As with the other example, you will need to download the WebApp files into a new directory. Use `sudo` where necessary. 
+
+```
+$ apt update && apt install -y python3 python3-pip curl
+$ mkdir ~/wott-webapp-auth-example
+$ cd ~/wott-webapp-auth-example
+$ curl -o app.py https://raw.githubusercontent.com/WoTTsecurity/agent/master/docs/examples/simple-webapp-auth/app.py
+$ curl -o requirements.txt https://raw.githubusercontent.com/WoTTsecurity/agent/master/docs/examples/simple-webapp-auth/requirements.txt
+$ pip3 install -r requirements.txt
+$ sudo python3 app.py
+
+```
+
+This will start your app on the server 127.0.0.1 at port 8080. You should receive a response like so:
+
+```
+* Serving Flask app "app" (lazy loading)
+* Environment: production
+  WARNING: Do not use for the development server in a production environment.
+  Use a production WSGI server instead.
+* Debug mode: Off
+* Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
+
+```
+
+## Accessing the WebApp
+
+Launch http://127.0.0.1:8080/ on your browser's device. Here, provided you have encountered no errors, you'll be prompted to enter a username and password. If you have followed the steps so far the username should be `username` and the password `password`. 
+
+With the correct details, you will be greeted with the message: 
+```
+Login successful. Hello from WoTT!
+
+```
+
+And that's it, you've set up basic HTML auth on a WebApp!
+
+
+## Common Errors
+
+**No module named flask**
+```
+File "/Users/user/dir/app_dir/app.py", line 1, in <module>
+    from flask import Flask
+ImportError: No module named flask
+
+```
+
+This can occur if the requirements file does not install properly. A simple fix is installing flask using `pip3 install flask` globally.
```

Comment:
`pip3 install flask` isn't the right solution here. It's `pip3 install -r requirements.txt`. The source of the problem is likely that the user wasn't in the virtualenv.

Reply:
Noted, thank you